EasyPost.Tests/PickupTest.cs

```diff
@@ -15,71 +15,99 @@ public void Initialize()
             _vcr = new TestUtils.VCR("pickup");
         }
 
-        private static async Task<Pickup> CreateBasicPickup()
```

Comment:
I'd like to keep the CRUD order of tests like we have in all other test files. Can we put the order back to:

1. create
2. retrieve
3. all
4. update
5. delete